### PR TITLE
Update schema.json with "validationSchema"

### PR DESCRIPTION
**Pull request by feki-rihab**

### CODE REVIEW

File: WeatherAlert/schema.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema#",
   "$schemaVersion": "0.4.1",
+  "validationSchema": "https://github.com/smart-data-models/dataModel.Weather/edit/master/WeatherAlert/schema.json",
```
Comment (flopezag) [inline WeatherAlert/schema.json]: No, validationSchema is not a key property of the schema.json but a new property that should be defined inside "properties": { }, with the corresponding type and description, where type should be a string, and description should make reference to the corresponding URL in the Model and description about the URL to the corresponding validation schema of the data model and you can put a default value of the link to the validationSchema.

**Comment by flopezag** [comment]
Missing to include the corresponding example in the example.jsonld with the new property

**Review by flopezag — CHANGES_REQUESTED** [review]

**Comment by albertoabellagarcia** [comment]
This attribute is not necessary because it is the$id of the schema. Indeed it can be retrieved with this function of the pysmartdatamodels package

import json
from pysmartdatamodels import pysmartdatamodels as sdm

dataModel, subject = "AgriSoil", "dataModel.Agrifood"

result = sdm.list_datamodel_metadata(dataModel, subject)
print(result["jsonSchemaUrl"])
with open("payload2.json", "w") as file:
    json.dump(result, file, indent=2)

**Comment by feki-rihab** [comment]
Would it make sense to add the validationSchema in these examples instead of the $id or what is the compromise. 
I did this PR to update the schema.json of the data models to be aligned with the waterverse fair principles requirements. 

Let me know how we can solve this, also @flopezag could you suggest what best to do in this situation?

**Comment by albertoabellagarcia** [comment]
I don't see the reason for this because you already have not only the link you need, but also the function for retrieving it. Could you be more precise about what FAIR principle is enforced with your contribution and not with the actual solution?

**Comment by feki-rihab** [comment]
Fernando can elaborate better on these fair principles I believe. He will take a look into this once he is available :) the update of the schema was upon his request.

**Pull request closed without merging**